relayer/cli/src/commands/query/channel.rs

```diff
@@ -41,32 +42,42 @@ impl QueryChannelEndsCmd {
         &self,
         config: &Config,
     ) -> Result<(ChainConfig, QueryChannelOptions), String> {
-        match (&self.chain_id, &self.port_id, &self.channel_id) {
-            (Some(chain_id), Some(port_id), Some(channel_id)) => {
-                let chain_config = config.chains.iter().find(|c| c.id == *chain_id);
-                match chain_config {
-                    Some(chain_config) => {
-                        let opts = QueryChannelOptions {
-                            port_id: port_id.parse().unwrap(),
-                            channel_id: channel_id.parse().unwrap(),
-                            height: match self.height {
-                                Some(h) => h,
-                                None => 0 as u64,
-                            },
-                            proof: match self.proof {
-                                Some(proof) => proof,
-                                None => true,
-                            },
-                        };
-                        Ok((chain_config.clone(), opts))
-                    }
-                    _ => Err(format!("cannot find chain {} in config", chain_id)),
-                }
-            }
-            (None, _, _) => Err("missing chain identifier".to_string()),
-            (_, None, _) => Err("missing port identifier".to_string()),
-            (_, _, None) => Err("missing channel identifier".to_string()),
-        }
+        let chain_id = self
+            .chain_id
+            .ok_or_else(|| "missing chain configuration".to_string())?;
```

Comment:
Can we change the error message so it's easier to identify the failing check?
```suggestion
            .ok_or_else(|| "missing chain id".to_string())?;
```